packages/repo/src/mst/mst.ts

```diff
@@ -404,6 +415,7 @@ export class MST implements DataStore {
   // returns entry at index
   async atIndex(index: number): Promise<NodeEntry | null> {
     const entries = await this.getEntries()
+    // TODO: shouldn't this error instead of returning null?
```

Comment:
hmmm why's that? 🤔

Reply:
Hrm, maybe just a different norm/expectation in Javascript land (vs python/go/rust).

Checking behavior of array indexing real quick, what about `undefined`?

Reply:
Yeah we call this in quite a few places where we expect the index to possibly be out of bounds

I think you're write that `undefined` is probably the better match in this case. Doesn't affect behavior at all:
```ts
  async atIndex(index: number): Promise<NodeEntry | undefined> {
    const entries = await this.getEntries()
    return entries[index]
  }
```